package.json

```diff
@@ -95,7 +95,7 @@
     "dependencies": {
         "prop-types": "^15.7.2",
         "react-select": "^3.0.4",
-        "react-transition-group": "^2.3.1",
+        "react-transition-group": "^4.3.0",
```

Comment:
I'm pretty sure that updating this dependency would cause issues and some components' code would require updates. But I don't remember exactly which components nor how to reproduce the issue atm 😅.
I should leave this out of this update.

Reply:
Closing this for now then. Will make another PR without react transition.